package.json

```diff
@@ -0,0 +1,88 @@
+{
+  "name": "riot-desktop",
+  "productName": "Riot",
+  "main": "src/electron-main.js",
+  "version": "1.5.5",
+  "description": "A feature-rich client for Matrix.org",
+  "author": "New Vector Ltd.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/vector-im/riot-desktop"
+  },
+  "license": "Apache-2.0",
+  "files": [],
+  "scripts": {
+    "mkdirs": "mkdir -p packages && mkdir -p deploys",
+    "fetch": "yarn run mkdirs && scripts/fetch-package.js",
+    "check": "scripts/check-webapp.js",
+    "start": "yarn run check && yarn install:electron && electron .",
+    "install:electron": "electron-builder install-app-deps",
+    "lint": "eslint src/",
+    "build:electron": "yarn check && electron-builder",
+    "clean": "rimraf webapp dist packages deploys"
+  },
+  "dependencies": {
+    "auto-launch": "^5.0.1",
+    "electron-store": "^2.0.0",
+    "electron-window-state": "^4.1.0",
+    "minimist": "^1.2.0",
+    "png-to-ico": "^1.0.2"
+  },
+  "devDependencies": {
+    "electron-builder": "^21.2.0",
```

Comment:
Assuming this experiment is a success, will it mean we can purge these Electron-only dev deps from `riot-web` repo?

Reply:
yep!